public/pages/Indices/components/CloseIndexModal/CloseIndexModal.tsx

```diff
@@ -36,13 +39,21 @@ export default function CloseIndexModal(props: CloseIndexModalProps) {
     return null;
   }
 
+  const showWarning = selectedItems.map((item) => filterByMinimatch(item as string, SYSTEM_INDEX)).filter((it) => it).length > 0;
```

Comment:
could we combine map and filter into one filter?
```suggestion
  const showWarning = selectedItems.filter((item) => filterByMinimatch(item as string, SYSTEM_INDEX))).length > 0;
```

Reply:
Thanks, I will change it.